### PR TITLE
Consistency test

### PR DESCRIPTION
**Pull request by wakamex**

### CODE REVIEW

File: sim.py [hunk 1]
```diff
@@ -5,94 +5,93 @@ class YieldSimulator(object):
...
         self.max_vault_apy = kwargs.get('max_vault_apy')
-        self.min_pool_age = kwargs.get('min_pool_age')
-        self.max_pool_age = kwargs.get('max_pool_age')
         self.base_asset_price = kwargs.get('base_asset_price')
         self.precision = kwargs.get('precision')
         self.pricing_model_name = str(kwargs.get('pricing_model_name'))
         self.trade_direction = kwargs.get('trade_direction')
         self.days_until_maturity = kwargs.get('days_until_maturity')
         self.num_trading_days = kwargs.get('num_trading_days')
-        self.rng = kwargs.get('rng')
```
Comment (dpaiton) [inline sim.py:31]: @wakamex can you put this back to how it was?

File: sim.py [hunk 2]
```diff
@@ -5,94 +5,93 @@ class YieldSimulator(object):
...
+            + f'init_pool_apy: {self.init_pool_apy}\n'
             + f'fee_percent: {self.fee_percent}\n'
             + f'init_vault_age: {self.init_vault_age}\n'
-            + f'init_vault_apy: {self.vault_apy[0]}\n'
-            + f'pool_age: {self.pool_age}\n'
+            + f'init_vault_apy: {self.vault_apy[0]}\n' # first element in vault_apy array
         )
 
     def run_simulation(self, override_dict=None):
+        self.trade_number = 0
```
Comment (dpaiton) [inline sim.py:94]: @wakamex can you rename this to run_trade_number

File: sim.py [hunk 3]
```diff
@@ -213,19 +234,22 @@ def update_analysis_dict(self):
...
+        self.analysis_dict['out_with_fee'].append(self.with_fee*self.base_asset_price)
+        self.analysis_dict['out_without_fee'].append(self.without_fee*self.base_asset_price)
+        self.analysis_dict['fee'].append(self.fee*self.base_asset_price)
+        self.analysis_dict['slippage'].append((self.without_fee_or_slippage-self.without_fee)*self.base_asset_price)
         self.analysis_dict['spot_price'].append(self.market.spot_price())
 
 
 class Market(object):
-    def __init__(self, x, y, g, t, total_supply, pricing_model, u=1, c=1, verbose=False):
+    def __init__(self, x, y, g, t, total_supply, pricing_model, u=1, c=1, verbose=False, floor_fee=0):
```
Comment (dpaiton) [inline sim.py:249]: @dpaiton make this a property of the specific market class

File: sim.py (hunk 2 above)
Comment (wakamex) [reply sim.py]: done

**Comment by wakamex** [comment]
[rename trade_number to run_trade_number](https://github.com/element-fi/analysis/pull/15/commits/3ef5cb54ce5ded39499b25c4af39828e9baf0470) is misnamed, it renames to run_trade_number in sim.py (hadn't hit save yet)

**Pull request merged**